src/scanner.cc

```diff
@@ -240,6 +240,90 @@ namespace {
     CLexState_END_OF_FILE
   };
   
+  /**
+   * Scans for block comment text. This scanner function supports nested
+   * block comments, so (* text (* text *) text *) will all be parsed as
+   * a single block comment. Also, multiple block comments separated
+   * only by spaces, tabs, or a single newline will be parsed as a single
+   * block comment for convenience; for example, the following is a
+   * single comment:
+   * 
+   *   (***********************)
+   *   (* text text text text *)
+   *   (* text text text text *)
+   *   (* text text text text *)
+   *   (***********************)
+   * 
+   * Although the following constitutes two separate block comments:
+   * 
+   *   (***********************)
+   *   (* text text text text *)
+   *   (***********************)
+   * 
+   *   (***********************)
+   *   (* text text text text *)
+   *   (***********************)
+   * 
+   * A block comment will also be emitted if EOF is reached.
+   *
+   * @param lexer The tree-sitter lexing control structure.
+   * @return Whether any block comment text was detected.
+   */
+  bool scan_block_comment_text(TSLexer* const lexer) {
```

Comment:
To be clear you can delete this function and the call to it. It's just the other stuff that has to stay, because the other stuff is how jlists and proofs know that they're seeing the start of a block comment.

Reply:
deleted